ui/component/livestreamComments/view.jsx

```diff
@@ -132,24 +167,38 @@ export default function LivestreamComments(props: Props) {
         <div className="livestream-discussion__title">{__('Live discussion')}</div>
         {superChatsTotalAmount > 0 && (
           <div className="recommended-content__toggles">
+
+            {/* the superchats in chronological order button */}
             <Button
               className={classnames('button-toggle', {
                 'button-toggle--active': viewMode === VIEW_MODE_CHAT,
               })}
               label={__('Chat')}
-              onClick={() => setViewMode(VIEW_MODE_CHAT)}
+              onClick={function() {
+                setViewMode(VIEW_MODE_CHAT);
+                const livestreamCommentsDiv = document.getElementsByClassName('livestream__comments')[0];
```

Comment:
I wonder if flow will complain that livestreamCommentsDiv could be undefined, so next line could break.

Reply:
It doesn't, just the two things I mentioned before

Reply:
nope you're right actually lol